pkg/runner/cluster_k8s.go

```diff
@@ -204,6 +204,33 @@ func (*ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow io.Wri
 
 	var gg errgroup.Group
 
+	// TODO: hack - this is the daemon, and not the user's local machine
+	workDir := filepath.Join(input.EnvConfig.WorkDir(), "results")
+	if err := os.MkdirAll(workDir, 0777); err != nil {
+		return nil, err
+	}
+
+	// TODO: hack - create the run output directory and write the runenv.
+	runDir := filepath.Join(workDir, input.RunID)
```

Comment:
@Stebalien note that this is different to the `local-docker` runner, which also adds the plan name. Not sure which one we prefer, but better be consistent.

Reply:
If we're going to put the results of each test in s3, I wouldn't use the plan name. I only used the plan name to make browsing `~/.testground/results/` easier.

Ideally, `testground run ...` would spit out the result location for the test.